en/advanced_config/ethernet_setup.md

```diff
@@ -35,7 +35,7 @@ To connect systems over Ethernet you need to configure them to run on the same I
 This might be done using a DHCP server to allocate addresses, or by manually configuring the addresses of each system on the network.
 
 There is no single "out of the box configuration" that we can provide that will necessarily work in your local network.
-Therefore as an example of the kind of configuration you might do, below we show how to set up the systems on an IP network with static addresses in the range `192.168.0.Xxx`, where PX4 has a statically allocated address `192.168.0.4` and the computer has address `192.168.0.1`.
+Therefore as an example of the kind of configuration you might do, below we show how to set up the systems on an IP network with static addresses in the range `10.41.10.Xxx`, where PX4 has a statically allocated address `10.41.10.2` and the computer has address `10.41.10.1`.
```

Comment:
@bkueng Just to be clear, this default IP address is set on boards, meaning that there is a default `net.cfg` somewhere, and below we're adding a value to overwrite this? The value is then used for setting things like where XRCE-DDS expects the default port and so on?

The reason I ask is that here and everywhere else in the doc the IP address is presented as a random example you might use. If it is a default then we probably should say that, because then presumably you don't have to do anything in the "PX4 Ethernet Network Setup" section?

So I was thinking you might say something like:

```suggestion
Therefore as an example of the kind of configuration you might do, below we show how to set up the systems on an IP network with static addresses in the range `10.41.10.Xxx`, where PX4 has a statically allocated address `10.41.10.2` (the default) and the computer has address `10.41.10.1`.
```

And then below in the "PX4 Ethernet Network Setup" section say something like - if you are happy to use this IP address you don't need to modify the configuration on PX4?

Reply:
Yes exactly, I was wondering if we should actually mention this is already the default.
The default behavior is also set to FALLBACK, which means it tries DHCP first. If you anyway want to use a static IP, you can directly set that and it gets faster initialized on startup.

Reply:
So PX4 will try read the IP from whatever DHCP server is configured. If it doesn't get an IP assigned it uses the default static IP. If you want to drop through that first step you have to explicitly define a config file with the default static IP (or some other value). Right?

Reply:
Correct

Reply:
Cool. Note added